PCbuild/_contextvars.vcxproj

```diff
@@ -0,0 +1,93 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Be aware that we recently removed this project file (it's compiled in `pythoncore.vcxproj`), so don't bring it back